website/docs/src/content/cloud/serverless-deployments.mdx

```diff
@@ -0,0 +1,234 @@
+```
+
+### CLI run options
+
+| Option      | Description                                     |
+| ----------- | ----------------------------------------------- |
+| `--input`   | JSON string with input data                     |
+| `--wait`    | Wait for workflow completion and return results |
+| `--output`  | Save results to a file (requires `--wait`)      |
+| `--project` | Specify the project name                        |
+
+## API endpoints
+
+Each workflow is exposed as an API endpoint:
+
+```
+https://api.gensx.com/v1/org/{org}/projects/{project}/environments/{environment}/workflows/{workflow}/run
+```
+
+- `{org}` - Your organization ID
+- `{project}` - Your project name
+- `{environment}` - The environment (defaults to "default")
+- `{workflow}` - The name of your workflow
+
+For example, if you have a workflow named `BlogWriter` in project `content-tools`, the endpoint would be:
+
+```
+https://api.gensx.com/v1/org/your-org/projects/content-tools/environments/default/workflows/BlogWriter/run
+```
+
+## Authentication
+
+All GenSX Cloud API endpoints require authentication using your GenSX API key as a bearer token:
+
+```bash
+curl -X POST https://api.gensx.com/v1/org/your-org/projects/your-project/environments/default/workflows/YourWorkflow/run \
+  -H "Authorization: Bearer your-api-key" \
+  -H "Content-Type: application/json" \
+  -d '{"prompt": "Tell me about GenSX"}'
+```
+
+### Obtaining an API Key
+
+To generate or manage API keys:
+
+1. Log in to the [GenSX Cloud dashboard](https://app.gensx.com)
+2. Navigate to Settings > API Keys
+3. Create a new key
+
+## Execution modes
+
+### Synchronous Execution
+
+By default, API calls execute synchronously, returning the result when the workflow completes:
+
+```bash
+curl -X POST https://api.gensx.com/v1/org/your-org/projects/your-project/environments/default/workflows/YourWorkflow/run \
+  -H "Authorization: Bearer your-api-key" \
+  -H "Content-Type: application/json" \
+  -d '{"prompt": "Tell me about GenSX"}'
+```
+
+### Asynchronous execution
+
+For longer-running workflows, use asynchronous execution by calling the `/start` endpoint:
+
+```bash
+# Request asynchronous execution
+curl -X POST https://api.gensx.com/v1/org/your-org/projects/your-project/environments/default/workflows/YourWorkflow/start \
+  -H "Authorization: Bearer your-api-key" \
+  -H "Content-Type: application/json" \
+  -d '{"prompt": "Tell me about GenSX"}'
+
+# Response includes an execution ID
+# {
+#   "status": "ok",
+#   "data": {
+#     "executionId": "exec_123abc"
+#   }
+# }
+
+# Check status later
+curl -X GET https://api.gensx.com/v1/executions/exec_123abc \
+  -H "Authorization: Bearer your-api-key"
+```
+
+### Streaming responses
+
+For workflows that support streaming, you can receive tokens as they're generated:
+
+```bash
+curl -X POST https://api.gensx.com/v1/org/your-org/projects/your-project/environments/default/workflows/YourWorkflow/run?stream=true \
```

Comment:
Do we actually support a `stream=true` query param?
```suggestion
curl -X POST https://api.gensx.com/org/your-org/projects/your-project/environments/default/workflows/YourWorkflow?stream=true \
```

Reply:
i know it works in the body of the request but have never tested the query param